pkg/config/analysis/msg/messages.yaml

```diff
@@ -560,6 +560,29 @@ messages:
   - name: "EnvoyFilterUsesRelativeOperation"
     code: IST0151
     level: Warning
-    description: "This envoy filter does not have a priority and has a relative patch operation set which can cause the envoyFilter not to be applied. Using the INSERT_FIRST option or setting the priority may help in ensuring the envoyFilter is applied correctly"
-    template: "This envoy filter does not have a priority and has a relative patch operation set which can cause the envoyFilter not to be applied. Using the INSERT_FIRST option or setting the priority may help in ensuring the envoyFilter is applied correctly"
-   
\ No newline at end of file
+    description: "This envoy filter does not have a priority and has a relative patch operation set which can cause the envoyFilter not to be applied. Using the INSERT_FIRST or ADD option or setting the priority may help in ensuring the envoyFilter is applied correctly"
```

Comment:
Capitalized EnvoyFilter.  It should look like usage in https://istio.io/latest/docs/reference/config/networking/envoy-filter/